### PR TITLE
Framework release v1.10

### PR DESCRIPTION
**Pull request by junkil-park**

### CODE REVIEW

File: metadata/v1.10/step_11_start_watching_google_jwks.json
Comment (thepomeranian) [inline metadata/v1.10/step_11_start_watching_google_jwks.json:1]: Discussion URL: https://github.com/aptos-foundation/AIPs/issues/331

File: metadata/v1.10/step_1_increase_max_txn_gas.json
Comment (thepomeranian) [inline metadata/v1.10/step_1_increase_max_txn_gas.json:1]: Discussion URL: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.10.0

File: metadata/v1.10/step_2_upgrade_framework.json
Comment (thepomeranian) [inline metadata/v1.10/step_2_upgrade_framework.json:1]: Discussion URL: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.10.0

File: metadata/v1.10/step_8_enable_max_object_nesting_check.json
Comment (thepomeranian) [inline metadata/v1.10/step_8_enable_max_object_nesting_check.json:1]: Discussion URL: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.10.0

File: metadata/v1.10/step_11_start_watching_google_jwks.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Start JWK consensus for Google",
+  "description": "JWK Consensus (AIP-67) for Google enables Google-based keyless accounts (AIP-61).",
+  "source_code_url": "https://github.com/aptos-foundation/mainnet-proposals/tree/main/sources/v1.10/step_11_start_watching_google_jwks",
+  "discussion_url": "https://github.com/aptos-labs/aptos-core"
```
Comment (thepomeranian) [inline metadata/v1.10/step_11_start_watching_google_jwks.json:5]: Discussion URL: https://github.com/aptos-foundation/AIPs/issues/331

File: metadata/v1.10/step_1_increase_max_txn_gas.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Increase max txn gas temporarily for framework upgrade",
+  "description": "Increase max txn gas temporarily for framework upgrade",
+  "source_code_url": "https://github.com/aptos-foundation/mainnet-proposals/tree/main/sources/v1.10/step_1_increase_max_txn_gas",
+  "discussion_url": "https://github.com/aptos-labs/aptos-core"
```
Comment (thepomeranian) [inline metadata/v1.10/step_1_increase_max_txn_gas.json:5]: Discussion URL: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.10.0

File: metadata/v1.10/step_2_upgrade_framework.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Multi-step proposal to upgrade mainnet framework to v1.10",
+  "description": "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.10",
+  "source_code_url": "https://github.com/aptos-foundation/mainnet-proposals/tree/main/sources/v1.10/step_2_upgrade_framework",
+  "discussion_url": "https://github.com/aptos-labs/aptos-core"
```
Comment (thepomeranian) [inline metadata/v1.10/step_2_upgrade_framework.json:5]: Discussion URL: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.10.0

File: metadata/v1.10/step_8_enable_max_object_nesting_check.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Max Object Nesting Check",
+  "description": "Enable the max object nesting check, preventing running out of gas due to ownership cycles.",
+  "source_code_url": "https://github.com/aptos-foundation/mainnet-proposals/tree/main/sources/v1.10/step_8_enable_max_object_nesting_check",
+  "discussion_url": "https://github.com/aptos-labs/aptos-core"
```
Comment (thepomeranian) [inline metadata/v1.10/step_8_enable_max_object_nesting_check.json:5]: Discussion URL: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.10.0

File: metadata/v1.10/step_11_start_watching_google_jwks.json
Comment (junkil-park) [reply metadata/v1.10/step_11_start_watching_google_jwks.json:1]: Updated!

File: metadata/v1.10/step_2_upgrade_framework.json
Comment (junkil-park) [reply metadata/v1.10/step_2_upgrade_framework.json:1]: Updated.

File: metadata/v1.10/step_11_start_watching_google_jwks.json (hunk 1 above)
Comment (junkil-park) [reply metadata/v1.10/step_11_start_watching_google_jwks.json:5]: Done.

File: metadata/v1.10/step_1_increase_max_txn_gas.json (hunk 2 above)
Comment (junkil-park) [reply metadata/v1.10/step_1_increase_max_txn_gas.json:5]: Done.

File: metadata/v1.10/step_2_upgrade_framework.json (hunk 3 above)
Comment (junkil-park) [reply metadata/v1.10/step_2_upgrade_framework.json:5]: Done.

File: metadata/v1.10/step_8_enable_max_object_nesting_check.json
Comment (junkil-park) [reply metadata/v1.10/step_8_enable_max_object_nesting_check.json:1]: Updated!

File: metadata/v1.10/step_8_enable_max_object_nesting_check.json (hunk 4 above)
Comment (junkil-park) [reply metadata/v1.10/step_8_enable_max_object_nesting_check.json:5]: Done.

**Review by igor-aptos — APPROVED** [review]

**Pull request merged**